dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -1120,4 +1120,17 @@ public Response getToolZip(@ApiParam(hidden = true) @Auth Optional<User> user,
         return Response.ok().entity((StreamingOutput)output -> writeStreamAsZip(sourceFiles, output, path))
             .header("Content-Disposition", "attachment; filename=\"" + fileName + "\"").build();
     }
+
+    @GET
+    @Timed
+    @UnitOfWork
```

Comment:
Looks like this can use that "read-only true" parameter above. 
Double-check whether this makes sense from a user perspective (this doesn't seem to check users like the zipDownload above)